### DISCUSSION TITLE
Is it possible to cancel all workflow runs for one specific commit?

### DISCUSSION THREAD

**Discussion by tobbi** [community/community#36712] [Actions]
I just pushed a commit and noticed I'd made a mistake that would definitely occur on all platforms I'm running in Actions.

Is it possible to cancel all workflow runs for that commit?

**Comment by jooy2** [comment]


**Reply by tobbi** [reply]
That only applies to one workflow. I want to cancel all workflows for one commit (preferably with one click)

**Comment by vitalyk-multinarity** [comment]
AFAIK, it's not possible.
But if you'll add proper tests to your workflows, the wrong code won't be deployed/published.

**Comment by ossanna16** [comment]
Hey there @tobbi and thank you for asking a good question! I'm glad you found our community 🙂
